janus_server/src/trace.rs

```diff
@@ -109,6 +139,82 @@ pub fn install_trace_subscriber(config: &TraceConfiguration) -> Result<(), Error
         );
     }
 
+    if config.otel_jaeger && config.otel_otlp.enabled {
+        return Err(Error::Other(anyhow::anyhow!(
+            "bad configuration, both Jaeger and OTLP OpenTelemetry layers cannot be enabled at the same time"
+        )));
```

Comment:
I don't think we need to introduce `anyhow` here. IMO the point of `anyhow` is to wrap arbitrary error types from any number of dependencies, but here we're working with an error that we create. Could you just define `Error::Other(&'static str)` and do `return Err(Error::Other("bad configuration (...)"))`